sunpy/coordinates/tests/test_frames.py

```diff
@@ -411,7 +413,11 @@ def test_skycoord_hpc(args, kwargs):
     sc = SkyCoord(*args, **kwargs, frame="helioprojective",
                   observer='earth', obstime="2011-01-01T00:00:00")
     # Test the transform to HGS because it will force a `make_3d` call.
-    hgs = sc.transform_to("heliographic_stonyhurst")
+    # astropy emits some incorrect warnings here, that will be fixed in a
+    # future release, so filter them out
```

Comment:
I think this comment needs elaboration on the precise "incorrect" warnings, because something may change and we may undesirably filter out "correct" warnings.